### PR TITLE
Time tag for paperwork

### PR DESCRIPTION
**Pull request by Birdtalon**
[time] tag replaced with current station time. One way conversion from pencode to text.

🆑 Birdtalon
add: adds [time] tag for paperwork
/🆑

### CODE REVIEW

**Comment by deep-fried-faceplant** [comment]
Is the timestamp like [sign] where it can't be forged? If it is, I have my reservations about this, but otherwise this is a nice QoL change.

**Comment by Birdtalon** [comment]
The timestamp will literally just translate the  current station time from when the "ok" button is pressed into text such as 13:05:42.

It could be "forged" by just writing "13:05:42" but I'm not sure what purpose that would serve?

**Comment by deep-fried-faceplant** [comment]
Forgery is an art, with an extreme emphasis on attention to detail. Minor things like these matter. Also, it just wouldn't make sense for the timestamp to be unique, it's not like you can tell what date a date was written on by looking at it.

File: code/__HELPERS/text.dm [hunk 1]
```diff
@@ -505,6 +505,7 @@ proc/checkhtml(var/t)
 		text = replacetext(text, "\[row\]",		"</td><tr>")
 		text = replacetext(text, "\[cell\]",	"<td>")
 		text = replacetext(text, "\[logo\]",	"<img src = ntlogo.png>")
+		text = replacetext(text, "\[time\]", "[gameTimestamp()]")
```
Comment (Fox-McCloud) [inline code/__HELPERS/text.dm]: add a `//TO DO` comment here, then we'll merge this.

**Pull request merged**